### ISSUE TITLE
Cannot deploy V1

### ISSUE THREAD

**Issue by maper822** [f5devcentral/f5-azure-saca#41]
When trying to deploy the following error is given. 
{'code':'DeploymentFailed','message':'At least one resource deployment operation failed. Please list deployment operations for details. Please see https://aka.ms/arm-debug for usage details.','details':[{'code':'Conflict','message':'{\\r\\n \\'status\\': \\'Failed\\',\\r\\n \\'error\\': {\\r\\n \\'code\\': \\'ResourceDeploymentFailure\\',\\r\\n \\'message\\': \\'The resource operation completed with terminal provisioning state 'Failed'.\\',\\r\\n \\'details\\': [\\r\\n {\\r\\n \\'code\\': \\'VMExtensionProvisioningError\\',\\r\\n \\'message\\': \\'VM has reported a failure when processing extension 'SetupAutomationScript'. Error message: \\\\\\'Enable failed: processing file downloads failed: failed to download file[0]: failed to download file: unexpected status code: got=404 expected=200\\\\\\'.\\'\\r\\n }\\r\\n ]\\r\\n }\\r\\n}'}]}

Have the files been relocated and the does the script need to be updated?

**Comment by davinmj** [comment]
The files have been relocated... I submitted a ticket with Microsoft and F5 to get it properly remapped.

**Comment by Mikej81** [comment]
V1 is dead and will not be updated anymore.

**Comment by davinmj** [comment]
> V1 is dead and will not be updated anymore.

Why is it sill offered?

**Comment by Mikej81** [comment]
Historical reference for original V1 deployments, and forked repos.

**Comment by davinmj** [comment]
But with the V2 3 tier only being an Alpha phase those of us in DoD that are required to do the 3 tier are we are just out of luck till the 3 tier is ready?

**Comment by Mikej81** [comment]
3 tier has been tested good by F5 and Microsoft. It is currently in use by several DOD customers. There are still chances it may not work with different requirements between IL5 and Azure Secret which is the reason for Alpha. 

All quickstarts should be used as a baseline, most customers may need to customize to their requirements.

**Comment by davinmj** [comment]
That's the thing... We are attempting to build in IL5 and unable to get V2 fully operational. That is why it was recommended to me to revert back to the V1.  It seems like the V1 was pulled offline before V2 was ready for operation in all Azure "IL" environments.

**Comment by Mikej81** [comment]
V1 hasn't actually worked since around January.  We redesigned everything into V2 with Microsoft in February / March.  Microsoft wanted to collapse everything to a single tier and that was the first use-case we were able to get completed.  Whomever recommended going back to V1 may have been misinformed.  If you are having issues with V2, then I recommend working with your account team and or Microsoft to find the issues.  

One thing to keep in mind, no repositories under DevCentral are supported.  The main contributor to SACA is me, and I am on PTO until July.  However, that does not mean that you cannot work with your account team to resolve whatever issues you are seeing.
